<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgCalculateProd</class>
 <widget class="QDialog" name="DlgCalculateProd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tab_indicators">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>351</width>
     <height>361</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="IndicatorsTab">
    <attribute name="title">
     <string>Indicators</string>
    </attribute>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>10</y>
       <width>166</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>311</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>NDVI dataset:</string>
     </property>
     <widget class="QComboBox" name="dataset_ndvi">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>275</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>321</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>Indicators to calculate:</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>291</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="1">
        <widget class="QLabel" name="label_26">
         <property name="font">
          <font>
           <weight>50</weight>
           <italic>true</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Trajectory is related to rate of change of  productivity over time</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_27">
         <property name="font">
          <font>
           <weight>50</weight>
           <italic>true</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Performance is a comparison of how productivity in an area compares to productivity in similar areas at the same point in time.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_28">
         <property name="font">
          <font>
           <weight>50</weight>
           <italic>true</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>State is a comparison of how current productivity in an area compares to past productivity in that area.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="indic_select_traj">
         <property name="text">
          <string>Trajectory</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="indic_select_perf">
         <property name="text">
          <string>Performance</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="indic_select_state">
         <property name="text">
          <string>State</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="TrajectoryTab">
    <attribute name="title">
     <string>Trajectory</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>321</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>Baseline period</string>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>60</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>End</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="traj_year_start">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="date">
       <date>
        <year>2000</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
      <property name="maximumDate">
       <date>
        <year>2030</year>
        <month>12</month>
        <day>31</day>
       </date>
      </property>
      <property name="minimumDate">
       <date>
        <year>1980</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
      <property name="displayFormat">
       <string>yyyy</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="traj_year_end">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="date">
       <date>
        <year>2015</year>
        <month>12</month>
        <day>31</day>
       </date>
      </property>
      <property name="maximumDateTime">
       <datetime>
        <hour>23</hour>
        <minute>59</minute>
        <second>59</second>
        <year>2030</year>
        <month>12</month>
        <day>31</day>
       </datetime>
      </property>
      <property name="minimumDate">
       <date>
        <year>1980</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
      <property name="displayFormat">
       <string>yyyy</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>321</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Trajectory indicator:</string>
     </property>
     <widget class="QComboBox" name="traj_indic">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>275</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>321</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Climate dataset:</string>
     </property>
     <widget class="QComboBox" name="traj_climate">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>275</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="PerformanceTab">
    <attribute name="title">
     <string>Performance</string>
    </attribute>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>2</y>
       <width>157</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Baseline period:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>End</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="perf_year_start">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="date">
      <date>
       <year>2000</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
     <property name="maximumDate">
      <date>
       <year>2030</year>
       <month>12</month>
       <day>31</day>
      </date>
     </property>
     <property name="minimumDate">
      <date>
       <year>1980</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
     <property name="displayFormat">
      <string>yyyy</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="perf_year_end">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="date">
      <date>
       <year>2015</year>
       <month>12</month>
       <day>31</day>
      </date>
     </property>
     <property name="maximumDateTime">
      <datetime>
       <hour>23</hour>
       <minute>59</minute>
       <second>59</second>
       <year>2030</year>
       <month>12</month>
       <day>31</day>
      </datetime>
     </property>
     <property name="minimumDate">
      <date>
       <year>1980</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
     <property name="displayFormat">
      <string>yyyy</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="StateTab">
    <attribute name="title">
     <string>State</string>
    </attribute>
    <widget class="QDateEdit" name="state_existdeg_early_year_start">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="maximumDate">
      <date>
       <year>2030</year>
       <month>12</month>
       <day>31</day>
      </date>
     </property>
     <property name="minimumDate">
      <date>
       <year>1980</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
     <property name="displayFormat">
      <string>yyyy</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>101</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Early epoch:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>70</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>End</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="state_existdeg_early_year_end">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="date">
      <date>
       <year>2010</year>
       <month>12</month>
       <day>31</day>
      </date>
     </property>
     <property name="maximumDateTime">
      <datetime>
       <hour>23</hour>
       <minute>59</minute>
       <second>59</second>
       <year>2030</year>
       <month>12</month>
       <day>31</day>
      </datetime>
     </property>
     <property name="minimumDateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>1980</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="displayFormat">
      <string>yyyy</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="state_existdeg_late_year_end">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>100</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="date">
      <date>
       <year>2015</year>
       <month>12</month>
       <day>31</day>
      </date>
     </property>
     <property name="maximumDateTime">
      <datetime>
       <hour>23</hour>
       <minute>59</minute>
       <second>59</second>
       <year>2030</year>
       <month>12</month>
       <day>31</day>
      </datetime>
     </property>
     <property name="minimumDate">
      <date>
       <year>1980</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
     <property name="displayFormat">
      <string>yyyy</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>101</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Late epoch:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>130</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>130</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>End</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="state_existdeg_late_year_start">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>100</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="maximumDateTime">
      <datetime>
       <hour>23</hour>
       <minute>59</minute>
       <second>59</second>
       <year>2015</year>
       <month>12</month>
       <day>31</day>
      </datetime>
     </property>
     <property name="minimumDateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2011</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="maximumDate">
      <date>
       <year>2015</year>
       <month>12</month>
       <day>31</day>
      </date>
     </property>
     <property name="minimumDate">
      <date>
       <year>2011</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
     <property name="displayFormat">
      <string>yyyy</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>201</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Existing degradation:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>201</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Emerging degradation:</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="state_emergedeg_base_year_start">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>210</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="maximumDate">
      <date>
       <year>2030</year>
       <month>12</month>
       <day>31</day>
      </date>
     </property>
     <property name="minimumDate">
      <date>
       <year>1980</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
     <property name="displayFormat">
      <string>yyyy</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>240</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>310</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>End</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>240</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>End</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="state_emergedeg_base_year_end">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>210</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="date">
      <date>
       <year>2015</year>
       <month>12</month>
       <day>31</day>
      </date>
     </property>
     <property name="maximumDateTime">
      <datetime>
       <hour>23</hour>
       <minute>59</minute>
       <second>59</second>
       <year>2030</year>
       <month>12</month>
       <day>31</day>
      </datetime>
     </property>
     <property name="minimumDateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>1980</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="displayFormat">
      <string>yyyy</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>280</y>
       <width>151</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Comparison period:</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="state_emergedeg_comp_year_end">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>280</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="date">
      <date>
       <year>2020</year>
       <month>12</month>
       <day>31</day>
      </date>
     </property>
     <property name="maximumDateTime">
      <datetime>
       <hour>23</hour>
       <minute>59</minute>
       <second>59</second>
       <year>2030</year>
       <month>12</month>
       <day>31</day>
      </datetime>
     </property>
     <property name="minimumDate">
      <date>
       <year>1980</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
     <property name="displayFormat">
      <string>yyyy</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="state_emergedeg_comp_year_start">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>280</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="date">
      <date>
       <year>2016</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
     <property name="maximumDateTime">
      <datetime>
       <hour>23</hour>
       <minute>59</minute>
       <second>59</second>
       <year>2030</year>
       <month>12</month>
       <day>31</day>
      </datetime>
     </property>
     <property name="minimumDateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2011</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="maximumDate">
      <date>
       <year>2030</year>
       <month>12</month>
       <day>31</day>
      </date>
     </property>
     <property name="minimumDate">
      <date>
       <year>2011</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
     <property name="displayFormat">
      <string>yyyy</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>310</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>141</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Baseline period:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>390</y>
     <width>158</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="button_prev">
      <property name="text">
       <string>Previous</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="button_next">
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QPushButton" name="button_calculate">
      <property name="text">
       <string>Calculate</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
